_locales/en/messages.json

```diff
@@ -32,6 +34,8 @@
   "This is a Greasemonkey user script.  Click install to start using it.": { "message": "This is a Greasemonkey user script.  Click install to start using it." },
   "Undo uninstall": { "message": "Undo uninstall" },
   "Uninstall": { "message": "Uninstall" },
+  "User script commands": { "message": "User script commands" },
+  "User script commands ...": { "message": "User script commands ..." },
```

Comment:
Since these are the exact same message, I don't think it's required to have both. Perhaps just adding the "..." in the code, while keeping the same translation message?

Maybe this is just an English thing and other languages would have different phrases, I don't know.

Reply:
`User script commands` is a heading of the sub menu and `User script commands ...` is a label of the top menu item.

I also don’t know, but I added the latter one in accordance with `New user script ...`.

Reply:
The text is the same, the usage may be not. Do you really want to split `Undo uninstall` into `Undo`, `SPACE` and `toLowercase`(`Uninstall`) in order to avoid same-text-translations? (NOT)

This may be exagerated, but the point is, these strings may be different in a translation.
Another example: Some languages conjugate differently. Some are associative.

Last example: Here I'd see even english "translations", that may differ:
Heading of sub menu may read `List of user commands`, while the main menu item could be `User script commands...`.

BTW: I'd prefer no `SPACE` before `...`, as seen in merely every other `Firefox` menu...

Reply:
> BTW: I'd prefer no SPACE before ..., as seen in merely every other Firefox menu...

+1

Reply:
What should we do about `New user script ...`?

Reply:
@esperecyan Submit a pull.